React/Views/RCTTextField.m

```diff
@@ -42,6 +42,25 @@ - (void)setText:(NSString *)text
   }
 }
 
+- (void)_setupPlaceholder {
```

Comment:
@nicklockwood - agreed, that's what I thought might be happening so I just wanted to make sure